### DISCUSSION TITLE
Issues in installing lazy vim on windows11

### DISCUSSION THREAD

**Discussion by Sam-at-code** [LazyVim/LazyVim#2973] [Q&A] [answered]
Hi i am beginner just installed the config files from the website but the plugins are not getting installed and nvim is in its default composition.where could i have gone wrong?

**Comment by dpetka2001** [comment]
First you should start by verifying you meet the [requirements](https://www.lazyvim.org/#%EF%B8%8F-requirements). Then follow the [installation steps](https://www.lazyvim.org/installation) for Windows. I don't use Windows, so can't help any further.

**Reply by Sam-at-code** [reply]
ya i meet the requirements and follow along the process still the issue persists

**Reply by dpetka2001** [reply]
What is the version of Neovim and git you're using?

**Reply by Sam-at-code** [reply]
nvim 0.9.5
git 2.44.0

**Reply by dpetka2001** [reply]
I just installed Neovim via scoop and installed on Windows and the install went smoothly as per the website's instructions. What does `nvim --version` show?

**Reply by Sam-at-code** [reply]
![Screenshot 2024-04-12 231452](https://github.com/LazyVim/LazyVim/assets/152985092/55c0423e-ec44-4b92-83d5-31403a32b440)

**Reply by dpetka2001** [reply]
Have you made any custom changes to your git config?

**Reply by Sam-at-code** [reply]
no

**Reply by dpetka2001** [reply]
Did you get any errors during installation?

**Reply by Sam-at-code** [reply]
no

**Reply by Sam-at-code** [reply]
![Screenshot 2024-04-13 000707](https://github.com/LazyVim/LazyVim/assets/152985092/1f625000-e5c6-4959-b1bf-3478a24e07c5)

**Reply by dpetka2001** [reply]
What command did you use to clone the repo?

**Reply by Sam-at-code** [reply]
git clone https://github.com/LazyVim/starter ~/.config/nvim

**Reply by dpetka2001** [reply, marked as answer]
![2024-04-12_21-59](https://github.com/LazyVim/LazyVim/assets/12776461/4c1e5b09-9b47-412e-baaf-eec015d36777)

Click on the Windows tab and follow the instructions there.

**Reply by Sam-at-code** [reply]
😵‍💫lol it was just due to this simple tab . Thanks man its working now

**Reply by Sam-at-code** [reply]
![Screenshot 2024-04-13 005705](https://github.com/LazyVim/LazyVim/assets/152985092/23e517f5-1e12-4df6-b24e-02d4fc604206)
![Screenshot 2024-04-13 005741](https://github.com/LazyVim/LazyVim/assets/152985092/0311559c-0ab9-440f-9c88-3ab31da9502a)
what to do about these failed plugins

**Reply by dpetka2001** [reply]
You need either `make` or `cmake` for that plugin to build. Google how to install them on Windows. Otherwise, just don't use the plugin.

**Reply by Sam-at-code** [reply]
ok thanks
